druid/src/widget/stepper.rs

```diff
@@ -42,26 +46,37 @@ pub struct Stepper {
 }
 
 impl Stepper {
-    pub fn new(
-        max: f64,
-        min: f64,
-        step: f64,
-        wrap: bool,
-        value_changed: impl Fn(&mut EventCtx, &mut f64, &Env) + 'static,
-    ) -> impl Widget<f64> {
-        Align::vertical(
-            UnitPoint::CENTER,
-            Stepper {
-                max,
-                min,
-                step,
-                wrap,
-                value_changed: Box::new(value_changed),
-                increase_active: false,
-                decrease_active: false,
-                timer_id: TimerToken::INVALID,
-            },
-        )
+    pub fn new(value_changed: impl Fn(&mut EventCtx, &mut f64, &Env) + 'static) -> Self {
```

Comment:
looking at this and at your example, i'm wondering if we even need _this_ argument? Unlike a `Button`, `Stepper` is mutating the data, and so this function feels like something that would only be used in special cases?

It's also possible that we don't need it at all?

Reply:
Right, it doesn't really seem necessary